tests/molecules/FeCO6_b3lyp_pyscf/det_vmcbatch_noj.in.xml

```diff
@@ -7,13 +7,13 @@
   <include href="FeCO6.wfnoj.xml"/>
   <random seed="17"/>
   <hamiltonian name="h0" type="generic" target="e">
-    <pairpot name="ElecElec" type="coulomb" source="e" target="e" physical="true"/>
-    <pairpot name="IonIon" type="coulomb" source="ion0" target="ion0"/>
     <pairpot name="PseudoPot" type="pseudo" source="ion0" wavefunction="psi0" format="xml">
       <pseudo elementType="C" href="C.BFD.xml"/>
       <pseudo elementType="Fe" href="Fe.BFD.xml"/>
       <pseudo elementType="O" href="O.BFD.xml"/>
     </pairpot>
+    <pairpot name="ElecElec" type="coulomb" source="e" target="e" physical="true"/>
```

Comment:
To be clear, these are reordered in order to trigger the bug, right?  The e-e and i-i potentials are constructed after the pseudopotential intentionally?

Reply:
Yes. The current develop fails on this.